qbraid/transpiler/converter.py

```diff
@@ -112,7 +122,7 @@ def transpile(
         path_details = _get_path_from_bound_methods(path)
         try:
             temp_program = deepcopy(program)
-        except RecursionError as err:
+        except (RecursionError, TypeError) as err:
```

Comment:
This was needed as pyqpanda3 mostly a wrapper around c++ types, and `deepcopy` gave en error:

```
TypeError: cannot pickle 'pyqpanda3.core.core.QProg' object
```